docs/docs/examples/llm/nvidia_nim.ipynb

```diff
@@ -4,16 +4,20 @@
    "cell_type": "markdown",
```

Comment:
<p>Looks like this information is repeated?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/13854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>